lib/Extension/LanguageServerCodeTransform/CodeAction/CreateClassProvider.php

```diff
@@ -32,7 +30,6 @@ public function __construct(private Generators $generators, private Parser $pars
     public function kinds(): array
     {
         return [
-            self::KIND,
             CodeActionKind::QUICK_FIX,
```

Comment:
@lumnn should this be `quickfix.createclass` or just `quickfix`?

Reply:
the `kind` is `<level1>.<level2>.<level3>` where `<level1>` is f.e. the value of `CodeActionKind::` - they define 1st level menus if you like.

just 

```
const KIND = CodeActionKind::QUICK_FIX;

// ...return [self::KIND];
```

and I don't know if we need more than that without actually seeing this in action.

Reply:
so this would be a menu like:

```
- quickfix:
    - create class
    - whatever
- refactor
    - extract method
    - whatever
   
```

Reply:
but without actually knowing how this information is used it's a bit like the blind leading the blind.